kolibri/core/assets/src/views/buttons-and-links/buttons.scss

```diff
@@ -152,4 +152,8 @@ a.button {
   &:hover:focus {
     outline: $core-outline;
   }
+  &:disabled {
```

Comment:
Let's add this disabled state to button examples in the style guide.

Reply:
![image](https://user-images.githubusercontent.com/1680573/44688917-9c17cb00-aa0a-11e8-929c-796017f8b97e.png)
?